pytest/testsuites/test_lis.py

```diff
@@ -0,0 +1,19 @@
+"""Runs 'LIS-Tests.xml' using Pytest."""
+import conftest
+import pytest
+from node_plugin import Node
+
+
+@pytest.mark.lisa(
+    platform="Azure", category="Functional", area="LIS_DEPLOY", tags=["lis"], priority=0
+)
+# @pytest.mark.deploy(setup="OneVM")
+@pytest.mark.connect("centos")
+def test_lis_driver_version(node: Node) -> None:
```

Comment:
It looks pytest discover test cases by names. So the test case writer need to do two things: 1) start with "test_" 2) has @pytest.mark.lisa
If they misses one of them, can it be found earlier?

Reply:
Pytest supports many ways of discovering tests. I am only using the most common way here. You can get [files, classes, functions, and change the glob pattern if you want, too](https://docs.pytest.org/en/stable/example/pythoncollection.html). You can even grab whole test modules from anywhere on the system and run `unittest` style tests. And if that's not enough, I've previously shared [their example of translating a YAML file _into a set of tests!_](https://docs.pytest.org/en/latest/example/nonpython.html#yaml-plugin)

As shown in the `criteria.yaml` proof-of-concept you can also verify that marks have been set as expected (and can do this with any hook that receives the `items` fixture, not just the one I used to modify the collection).